libraries/core/src/main/java/com/paritytrading/philadelphia/FIXConnection.java

```diff
@@ -153,15 +170,6 @@ public FIXConnection(CHANNEL channel, FIXConfig config, FIXMessageListener liste
         this.currentTimestamp.setTimestampMillis(this.currentTime);
     }
 
-    /**
-     * Get the underlying socket channel.
-     *
-     * @return the underlying socket channel
-     */
-    public CHANNEL getChannel() {
```

Comment:
This change will break client code, but is easy to work around by passing the channel by other means (DI, static, threadlocal are plausible workarounds work if one needs to upgrade, but cannot afford to refactor right now).

Reply:
My personal view is that I'd rather go without. This improves the encapsulation of the class, and if someone wants access to the channels there are many easy ways of doing it (i.e. the class that created the channels is a better place to look them up from).

Reply:
Alright.